pkg/volume/cinder/provisioner/iscsi.go

```diff
@@ -25,6 +25,7 @@ import (
 )
 
 const iscsiType = "iscsi"
+const initiatorName = "iqn.2018-01.io.k8s:a13fc3d1cc22"
```

Comment:
Can you help me understand why this name is hard coded?

Reply:
@hogepodge You mean rather than say something like generating something unique per cluster or making this a variable?  I guess a variable with a default to ensure they match might be a good option here.

Reply:
This stems from the way we handle volume attachment in the standalone cinder provisioner.  After we create a volume, the provisioner calls the os-initialize_connection API in cinder to retrieve the connection information for the volume.  Then, we translate that information into a native kubernetes PV if it uses a backend we support (currently iscsi and rbd).  When the volume needs to be attached to a node kubernetes handles this using the information we populated in the PV.  Since at provision time we can't know which node will be attached we hard-code an initiatorName and ask kubernetes to always use this initiatorName (instead of the host-specific one).  This has the effect of bypassing initiatorName-based controls that might be used by the storage (eg, xtremio).  The advantage is that the administrator does not need to manually edit /etc/iscsi/initiatorname.iscsi on each node in the cluster in order to use this provisioner.